### ISSUE TITLE
Alexa suddenly started saying "device not responding"

### ISSUE THREAD

**Issue by octoplayer2** [bwssytems/ha-bridge#1125]
HA bridge has been working perfectly for years+, V4.5.5 on a Rasp PI.
Last night it suddenly stopped responding to voice requests, Alexa says for any device that "the device is not responding".
I "forgot" all devices on Alexa, and rediscovered them, so that is talking to the HA Bridge.
I can control devices from the test buttons on HA bridge, so that is working.
I have restarted the router, Alexa, and PI, but still get the problem.

There was a warning in the Bridge log about the UPNP IP not having a valid device on it - spurious as the IP was correct and for the PI that Bridge is running on.  I reset value to 0000, reinitialised and it has updated back to the same IP, but without the error in the log this time.

The only thing that changed on the network was that I added a new HP wifi printer to the network yesterday, but the IP for that is correct, and it is turned off. I cannot see any changes in the router that that could have made.

Any suggestions welcome...

**Comment by bwssytems** [comment]
@octoplayer2 Let's try rebooting your raspberry pi and try again. There have been issues in the pi networking I have seen in the past and a reboot may correct this.

**Comment by octoplayer2** [comment]
Thanks for response... I Rebooted PI (again), but Alexa still says device not responding.   :-(

**Comment by mcwieger** [comment]
I second this issue. On Synology DS414.

**Comment by bwssytems** [comment]
Alright then, You will need to turn on traceupnp and turn on the debug for the HueMulator class, then do a discovery and then try to activate a device. Then post the log. Also, include IPs of the echo

**Comment by mondieumathieu** [comment]
Same thing here (germany) , worked for years, Since a couple of days. "device not responding" (Gerät antwortet nicht) . I used always port 8080, have 2 echos gen1 one echo gen2. Also when I delete a device I can find it only with the android app again, not in windows browser. Should I try another port? Regards, Maddes

**Comment by octoplayer2** [comment]
Ok, I had meanwhile replaced jar with 5.3.0, just in case, and also noticed that upnp was disabled in the router. Enabled that and rebooted router. No change.
I enabled upnp trace and Huemulator. ran Discovery and requested a device activation - no change. However, as far as I can see, the discovery reported back to all 3 Alexa devices, but the activation request has not appeared in the log (attached).

Pi IP is 192.168.1.114, the alexas are .111, .127 and .128.

[09.txt](https://github.com/bwssytems/ha-bridge/files/3584600/09.txt)

**Comment by bwssytems** [comment]
@octoplayer2 So, can you run the bridge on port 80 and not 8080? In another thread, this worked for another where it was fine before and now it is not. Maybe a new echo SW update.

**Comment by sneaker8650** [comment]
Got the same Issue. Alexa could no longer find any Devices which worked for months without problems.

Updated to the newest Version of ha-bridge. Still the same Error. Changed the webinterface port to 80, like suggested and its working again. :)
I´m using an Echo 1st gen and an 3rd gen Echo btw.

**Comment by octoplayer2** [comment]
I've have edited the Web Server Port under Bridge Control, and reinitialised, but it keeps reverting to 8080. Is it set somewhere else?

**Comment by octoplayer2** [comment]
Ahh, just found an old Argument in the .service file, deleted the DServerPort option there and will try again...
Took a few goes, including having to set Web port one last time and save, plus Forgetting all devices in Alexa and rediscovering, but it is working again.  I appear to have duplicates now of all devices - one as "Philips Hue White Light" and one as "Royal Philips Electronics smart device"; will go and delete the Hue versions

Thanks all for help.  Definitely looking like some change in Alexa.

**Comment by Maarten112** [comment]
I had the same issue since a few days. 

Solution: Let HA bridge run on port 80. 

Amazon must have pushed a new firmware to our devices, that only will communicate with ha bridge over port 80.

**Comment by octoplayer2** [comment]
Update: the hue white light version of the devices are rediscoverable; Alexa does not give its usual warning of "two devices with that name", but seems to pick one at random. It then gives a "cannot communicate with the provider" message.  Manually forgetting them seems to work but early days...

**Comment by mondieumathieu** [comment]
> Same thing here (germany) , worked for years (v5.2.1), Since a couple of days. "device not responding" (Gerät antwortet nicht) . I used always port 8080, have 2 echo dot gen 1 or 2, one echo dot gen 3. Also when I delete a device I can find it only with the android app again, not in windows browser. Should I try another port? Regards, Maddes

Thank you guys, Changed to port 80. Works again

I had to change 
habridge.service
sudo nano /etc/systemd/system/habridge.service
and 
habridge.config
sudo nano /home/pi/habridge/data/habridge.config

**Comment by mcwieger** [comment]
Hmm, I'm running another service on port 80 already, so habridge won't start with that option. Trying to start with another webserver, using:

`nohup /volume1/@appstore/Java8/j2sdk-image/bin/java -jar -Dconfig.file=/volume1/web/ha_bridge/data/habridge.config -Dserver.ip=192.168.2.111 -Dserver.port=8085 /volume1/web/ha_bridge/ha-bridge-5.3.0.jar`

Response is `The upnp config address, 192.168.2.111, does not match any known IP's on this host. Using default address: 192.168.2.201`

So how can I start it to run at a different web server address?

**Comment by dinkybluebug** [comment]
same for me.... Im running on port 80 already. Everything stopped working today saying 'device not responding' - my habridge in browser works at http://192.168.1.12/#!/

Ive forgotten all the devices, and now when I discover them again, it cant find them

**Comment by gohamstergo** [comment]
sound like amazon screwed the pooch. take a minute had contact them to submit a ticket to the 'technical team.' I just did. The more people that do, the better chance they'll pay attention.

**Comment by mondieumathieu** [comment]
> Hmm, I'm running another service on port 80 already, so habridge won't start with that option. Trying to start with another webserver, using:
> 
> `nohup /volume1/@appstore/Java8/j2sdk-image/bin/java -jar -Dconfig.file=/volume1/web/ha_bridge/data/habridge.config -Dserver.ip=192.168.2.111 -Dserver.port=8085 /volume1/web/ha_bridge/ha-bridge-5.3.0.jar`
> 
> Response is `The upnp config address, 192.168.2.111, does not match any known IP's on this host. Using default address: 192.168.2.201`
> 
> So how can I start it to run at a different web server address?

here is information about a "reverse proxy":
https://github.com/openenergymonitor/emonpi-ha-bridge
may be this can help?

**Comment by gsebil08** [comment]
Hello, same trouble for me. Was working fine for now months, and since this morning "the device is not responding".
Ha-bridge was on port 9898, put back Ha-bridge on port 80, and it is working perfectly.
Could you please check this trouble, I don't like having things running on this port...

**Comment by vco1** [comment]
Same issue over here. Like @dinkybluebug ha-bridge (5.3.0 but it started with 5.2.1)  is, and has always been, running, on port 80. So I'm a bit lost on what to change - if anything - to get it working again.
Searching for new devices in the Alexa app doesn't yield any results.
Any suggestions to get it working again would be very much appreciated as I really like the convenience of switching on the lights with Alexa.

**Comment by tobiasHaecker** [comment]
Good morning,
also the same situation on my side.
This morning I had a call with the Amazon support team. The guy there told me to reset my echo and to unplug and wait for 3 minutes.....
He told me that there were no software changes at Amazon between the 6th and the 7th.
So what next?

**Comment by gohamstergo** [comment]
well your amazon guy is wrong. there were definitely software changes somewhere between the 4th-7th (I went out of town). I came home and the light bar now rotates blue when searching devices and the light bar also turns off between you talking and alexa executing a command. It didn't do that before the 4th.

**Comment by vco1** [comment]
I'm not sure how up-to-date Amazon's firmware pages are usually, but the firmware versions for my Dot and Echo are (much) higher than the versions mentioned on the Amazon website. Perhaps they accidentally pushed the wrong firmware?
Unrelated to this issue, but I also noticed a loud click on my Dot before Alexa answers. Which wasn't there before afaik.

**Comment by Maarten112** [comment]
> well your amazon guy is wrong. there were definitely software changes somewhere between the 4th-7th (I went out of town). I came home and the light bar now rotates blue when searching devices and the light bar also turns off between you talking and alexa executing a command. It didn't do that before the 4th.

yes, the amazon guy clearly didn't know what he was talking about. 

What I Noticed:
- it seems amazon pushes the firmware update based on date of registration for your alexa account
- I am a pretty early adapter, and got the problems about 6 days ago. Two friends that bought their items much later don't have any problems (yet). I'm sure however they will run into the same issue.
- In the alexa app you can see what software version's your echo (dot's) are running. I am running a different and higher numbered version than my friends.
- however...both my echo dot's running another software version......not sure how this works.



> Same issue over here. Like @dinkybluebug ha-bridge (5.3.0 but it started with 5.2.1) is, and has always been, running, on port 80. So I'm a bit lost on what to change - if anything - to get it working again.
> Searching for new devices in the Alexa app doesn't yield any results.
> Any suggestions to get it working again would be very much appreciated as I really like the convenience of switching on the lights with Alexa.

Maybe try this:

1. delete all your smart home devices in the alexa app
2. run HA bridge on port 80
3. open HA bridge gui, click on "renumber devices"
4. Let alexa discover new devices

**Comment by gsebil08** [comment]
Worked for me:
1. Run HA-Bridge on port 80
2. Delete all devices on Alexa app
3. Discover devices from Alexa

On the first attempt I didn't delete the devices on Alexa app, they just went back from "unreachable" state to "ok" on Alexa app after changing port to 80. But a few hours later they went to "this object does not support this command" state (or something like that).
Then I deleted them from Alexa App, and discovered them again. Everything is now back to normal.

**Comment by vco1** [comment]
@gsebil08 Deleting etc. is not working for me. What options for discovery in the app did you use? Light - Other, or Philips Hue and a V1 or V2 Hub version?

**Comment by gsebil08** [comment]
@vco1 I select "other". I have a V2 echo dot. ha-bride-5.3.0.jar running.

**Comment by dinkybluebug** [comment]
So mine stopped working, I got it working, and now today it’s stopped working again. What’s going on?

**Comment by gohamstergo** [comment]
Amazon sucking is what. My Harmony hub still connects to ha-bridge just fine and I haven't changed a thing. It's just Alexa being stupid.

**Comment by techlemur** [comment]
Had the same problem, running a reverse proxy with apache2 on port 80 fixed it.  

See https://github.com/bwssytems/ha-bridge/issues/349 for instructions.

**Comment by vco1** [comment]
Just changing the port to 80 doesn't work for me. Does ha-bridge need to be available from the outside as well? I.e. do I need to forward the port from my router?

**Comment by techlemur** [comment]
Shouldn't need to, your echo's just need to be on the same network.  Did you forget the devices and re-discover after switching to port 80?

**Comment by tuicemen** [comment]
Seems the forget all devices option fails to work in the Alexa app for some. https://github.com/bwssytems/ha-bridge/issues/1133#issuecomment-532270485
Forgetting each device separately then rediscovering is what worked for me.

**Comment by stephenmhall** [comment]
I reported the problem to Amazon and got a request for a chat for more details, before I got round to it everything started working again including a name change I had tried that had not shown up in a discovery attempt the day before.

**Comment by alzzy** [comment]
I have same issue. I tried to forget each device individually and then rediscover. Only discovered the echo dots but no bridge devices. What else can I try. Can't use port 80.

**Comment by stephenmhall** [comment]
alzzy, what is your software version? mine is currently 641571120 I wonder with it just starting to work again after reporting the issue, and it magically discovering a renamed device without me asking, if tech support pushed a change that made it functional again. Mine is a full echo device.

**Comment by alzzy** [comment]
mine says version 2785574276 and is echo dot v3. I was on the phone with
tech support for about an hour this morning. I told him that a lot of folks
are reporting this issue on the web. We tried the usual stuff and by the
end of the hour he seemed convinced that it is a real issue and told me he
would research the situation and call me back. Have not heard back yet.

**Comment by alzzy** [comment]
Amazon support could not identify an issue except possibly that I am using an older version of the HA-bridge (4.5.0). Do I need to use the current version of the bridge to fix the fact that Alexa doesn't discover devices any more or should the version I am using still work?

**Comment by stephenmhall** [comment]
I'm on 5.3.0 all working at the moment, no port change.

**Comment by jmar83** [comment]
It has also a problem if you run it on a "special" IP address (maybe due to conflicts with port 80 on your primary one?): https://github.com/bwssytems/ha-bridge/issues/1135 

That can be a reaon too..

**Comment by alzzy** [comment]
I am using ha-bridge v4.5.0. Could that suddenly be the problem? It was working fine til 3 or so days ago.

**Comment by tuicemen** [comment]
@alzzy using an old version of HA-Bridge certainly won't  help and updating to the newest wont hurt. The issue is the Alexa app Amazon messed up with the last release. Most software hue bridges have the same issue. I believe Amazon has a fix they are slowly rolling out as my city system was now working but now it is.
Or it was my messing with it that got it working again. Once I got all devices forgoten in the app I told Alexa to discover I did not use the app to discover devices.

**Comment by gsebil08** [comment]
There was definitely an update on Alexa side. Now I have trouble with colors...
`Alexa, turn light in green` => `light is purple`
When I select the color directly from Alexa App, same trouble. I have to select Green to have Purple...
Why on Alexa side? Because when I test RGB color from ha-bridge UI, everything is ok.
Am I the only one facing this problem?

**Comment by ml9907** [comment]
Erease all existing hue devices from alexa, switch habridge to port 80 and rediscover devices. This helped me.

**Comment by jasonesmith12** [comment]
Mine stopped working to about 3 weeks ago. It's got to be changes that Amazon has made. I have used HA Bridge for years to automate all my X10 devices and even have a popular video on YouTube and recruited hundreds of people to using HA Bridge as a side effect. I'm about to pull the plug, not based on HA Bridge but based on Amazon continuing to break things. I think I'm going to switch over to all tp-link devices.

**Comment by alzzy** [comment]
I kept v4.5.0 of the bridge but changed its port to 80. Now Alexa finds devices and they are working normally. Amazon has done something that now requires bridge to be on port 80.

**Comment by ryankne** [comment]
Deleting all devices in the Alexa app, renumerating in ha bridge , and discovering again worked for me , I always had it on port 80 , my last issue was the dim command , was replying with not responding so I created a custom routine of “Alexa dim the lights “ to set the level to 50% (which runs scripts on my server)  , my ha-bridge is probably a 3 year old version though , lol

**Comment by quadhammer** [comment]
I can't use port 80 because my ISP blocks it.  So frustrating.  It works on port 8092 with an Echo V1, but not with dot v2.

Is there a way to run the habridge on port 80, but have the internet side on something like 8092?

**Comment by jasonesmith12** [comment]
Quadhammer, re: forwarding port to port oh, you might be able to do this on the local machine that's running h a bridge. If it's a Windows machine, you may be able to do something in the hosts file. it's not meant for forwarding port to port alone, but it looks like that I found someone that has done it with a combination of the hosts file and netsh. I don't know how deep you want to go into this. My experience when I saw Amazon break this before is that they finally fixed it within a month so the easiest remedy might be just to wait. However, if you want to try what you described, here is a link to someone who figured out how to forward a port to another port. Go down to the part where netsh is mentioned. https://stackoverflow.com/questions/8652948/using-port-number-in-windows-host-file

**Comment by bwssytems** [comment]
@alzzy I don't know how much pull I have with amazon since I am just an open source project. The community has gotten results out of Logitech when the harmony hub tried to turn off the interface that the ha-bridge was using. YMMV

**Comment by morgi23** [comment]
Hi,
How do I change the bridge port from 8080 to 80? I’m using latest Ha Bridge 5.3.0 on a raspberry pi. 
Thanks!

**Comment by bwssytems** [comment]
https://github.com/bwssytems/ha-bridge/blob/master/README.md

**Comment by quadhammer** [comment]
Thank  you, @jasonesmith12 

I'll try fiddling with the hosts files.

I've emailed Amazon to ask them to reinstate HUE usage on other ports besides exclusively 80, and I'd encourage everyone else in this thread to do the same:

https://alexa.amazon.com/spa/index.html#help/email

has a section in the webpage to quickly do it.

**Comment by morgi23** [comment]
Changing to port 80 from “Bridge control” tab is not working for me. It keeps going back to port 8080. 
I might doing something wrong here...

**Comment by quadhammer** [comment]
> Changing to port 80 from “Bridge control” tab is not working for me. It keeps going back to port 8080.
> I might doing something wrong here...

Same here.  Try stopping the server and starting it with the -Dserver.port argument instead.
In Windows the .bat file looks something like this:
@echo off
cd c:\echobridge
java -jar -Dserver.port=80 "C:\echobridge\ha-bridge-5.3.0.jar"
exit

The readme.md above will have more info.

**Comment by icibt** [comment]
I faced the same issue. First thing I did was I changed the port to 80 but to no avail. After that I switched the logging level to DEBUG and actually saw that Alexa was actually trying to add new user (weird?) but was hitting an error **"user add toContinue was false, returning not authorized"** in the HueMulator.java class. There were some **if** conditional statements in this class and they had something to do with the **Use Link Button**. So I set the **Bridge Control -> Update Security Settings -> Use Link Button** to false, Updated the settings and Alexa immediately found all the devices. So you might try that.

**TL;DR;** Try setting **Use Link Button** to **false** in **Bridge Control -> Update Security Settings -> Use Link Button** 

I am using (Ha-Bridge 5.1.0)

**Comment by gohamstergo** [comment]
Wiped all devices from my Echo and tried without link button. no joy. no log activity from my dot in the debug logs either.

**Comment by morgi23** [comment]
Ok, managed to solve this problem.
What worked for me was to change port 8080 to port 80.
Instructions for newbies like me:
1. Install Putty
2. SSH into the raspberry pi (you need to enter the the IP address of the raspberry pi)
3. Enter user name and password
4. Enter this command: sudo nano /etc/systemd/system/habridge.service
5. Edit the file : change port 8080 to 80, and press ctrl-X, then Y.
6. Enter this command: sudo nano /home/pi/habridge/data/habridge.config
7. Edit the file : change port 8080 to 80, and press ctrl-X, then Y.

Done :)
It worked for me without rediscover the devices, but if it doesn't work, try to forget and rediscover all alexa devices from the alexa app.

**Comment by gohamstergo** [comment]
The issue for most people is that they can't use port 80. It's why we weren't using it to begin with as other services are already using it.

**Comment by morgi23** [comment]
> The issue for most people is that they can't use port 80. It's why we weren't using it to begin with as other services are already using it.

What kind of services?

**Comment by mcwieger** [comment]
For Synology users services like Photo Station. Issue for me is that I've seen fragmented tips and hints to create or update files with certain info to run a reverse proxy, but no step-by-step guide. I mean, I can update a file by pasting text in it, but no one has been able to say which file and/or where they are located. I lack the Apache/Nginx knowledge to know where everything is located (plus it seems that on every version of Linux config files are located in different places).

**Comment by gohamstergo** [comment]
> > The issue for most people is that they can't use port 80. It's why we weren't using it to begin with as other services are already using it.
> 
> What kind of services?

read the other comments

**Comment by CedGexx** [comment]
In my case, replacing jetty's port to 80, renumbering the devices in HA Bridge, forgetting all devices and discovering again didn't help much. After enabling and checking the log,  I saw that they call made by alexa is not with the right ID. For example, a call to device ID 122 provides these logs:
```
2019-10-04 04:56:11,351 [qtp6470313-72] INFO  com.bwssystems.HABridge.hue.HueMulator - Tracestate: hue state change requested: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx from 192.168.x.xx body: {"on":true}
2019-10-04 04:56:11,353 [qtp6470313-72] WARN  com.bwssystems.HABridge.hue.HueMulator - Could not find device: 14 for hue state change request: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx from 192.168.x.xx body: {"on":true}

2019-10-04 04:57:22,421 [qtp6470313-70] INFO  com.bwssystems.HABridge.hue.HueMulator - Tracestate: hue state change requested: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx from 192.168.x.xx body: {"on":false}
2019-10-04 04:57:22,422 [qtp6470313-70] WARN  com.bwssystems.HABridge.hue.HueMulator - Could not find device: 14 for hue state change request: xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx from 192.168.x.xx body: {"on":false}
```
This is the old ID (i.e. 14). The only solution I have is to call each devices via Alexa to trigger an entry in the log and note the old ID, and then replace them in the ./habridge/data/devices.db
By doing so, it works as before except that some of the "switches" I have created are not found by Alexa...
Progressing slowly to put everything back up again.

**Comment by heffneil** [comment]
Man this is odd.  There is trouble here for some strange reason.  I mean between this and the port 80 thing tack on the inability for many of us to add new devices it sounds like we should be nervous.  I don't want to forget devices and pray I don't have problems communicating with what I have.... Fingers crossed this gets better or figured out soon!

**Comment by audiofreak9** [comment]
Also, what is the setting for the "UPNP IP Address" input on the Bridge Control page?

**Comment by e-bits** [comment]
Hi,
i'm facing exactly the same issue on my side. Running ha-bridge on Synology NAS. I try to connect the HUE App to ha-bridge. This is only working if i'm running on Port 80. HUE App can find the Bridge and i can successfully add the emulated devices. If i try the same running on Port 8080 this won't work.

So my question is this issue really related to an Alexa Firmware upgrade? Because if HUE App can't find the ha-bridge running on Port 8080 I guess Alexa can't find the devices either.

I'm pretty new to ha-bridge and probably I'm mixing things up. However, I would be interested to hear your opinion.

Thanks. Cheers e-bits

**Comment by bwssytems** [comment]
Make sure you have read all about the networking routes for your synology. Also, if you are using docker, there are even more things you will need to do. Search the issues here.

**Comment by Dr-know** [comment]
> 
> 
> Make sure you have read all about the networking routes for your synology. Also, if you are using docker, there are even more things you will need to do. Search the issues here.

Alexa not finding devices on my HA-bridge running in docker image on Synoloy !
I can not change HA-bridge to port 80 --> this is thi gui port for Synology and they do not let me change it ..

Where is this "networking routes for synology with docker" ??

Ps, are there plans to also include a sonos api helper ? like Fibaro and Harmony ? 
i no users a other docker image for this sonos API ..

**Comment by jimkernsjr** [comment]
Hi All...
First a couple points to the noobs:
Your ISP blocking port 80 has NOTHING to do with ANYTHING here.  YOU DO NOT FORWARD ports to the WAN (Internet) to use this software!  When I see comments like "my ISP blocks 80" and thumbs up on that comment I have concern.  My response is good - it's protecting you from yourself.  The port forwarding comes into play when you are running this in a docker or VM and have other services running on that port on the host machine.  It does NOT mean forwarding 80 to the internet.  Please read the readme more carefully, and please understand what you are doing when you forward ports to the internet before your expose your NAS or other device to attack.

Also the windows hosts file (or linux for that matter), has nothing to do with ports!  It's for host name to IP recognition only.  Tinkering with the hosts will NOT do anything to help with this software - it doesn't work by hostname.

That being said, this software must be run on 80 for it to work reliably with any echo device >gen 2 - there are several methods to accomplish that.  If you are new and you really dont know how to forward ports to VM's/Docker/etc, a quick fix can be to run the software on a Raspberry Pi (DO NOT install Apache/Web server).  Even a $10 Pi Zero works it seems.

I've also been having a similar problem with "not responding", yet it still works.  If I ask to turn on the lights, it does, then says after a couple seconds "not responding".  Other times it says "OK".    I'm on 5.3.0.  Anyone else run into this?  Is there a timeout  somewhere?  
I killed all devices and rediscovered - nothing.

**Comment by jimkernsjr** [comment]
**Update**  OFF works ok, but ON gives "blah blah blah" appears to be malfunctioning.  Of course there is no difference pressing OFF and ON on the web GUI - both respond promptly as successful.  So somehow between the bridge and Alexa, there seems to be a wierd issue.  Turning logs to DEBUG doesnt cause anything to jump out at me.

**Comment by audiofreak9** [comment]
@jimkernsjr thanks for taking the time to explain the incorrect assumptions, I couldn’t get past my facepalm to reply.

All, Phillips Hue uses port 80, Amazon is aligning with Phillips Hue.  This is also why the official HUE apps won’t discover your bridges that don’t use port 80.  I doubt Amazon will undo this software change.

**Comment by Shinjo01x** [comment]
Even if this thread is old, I ll try my luck. 

My habridge was working fine on 8080 until i Need to reinstall ist yesterday. I ve tried everything, but cant get alexa find any devices. I guess ist caused to the port change and I Need to use port 80. But I rund my Bridge on a synology nas, so cant use port 80. 

Does anyone have a solution? and can maybe guide me to get it work again?

**Comment by bwssytems** [comment]
PLease try the new RC4 release

**Comment by Kammi61871** [comment]
on pi's u can add a new ip on your network adapter works on eth0 and wlan0
[code]
sudo ifconfig wlan0:0 192.168.0.144 netmask 255.255.255.255
[code]
so u can make multiple port 80 on 1 device

output sudo ifconfig

[code]
pi@raspberrypi:~ $ sudo ifconfig
eth0: flags=4099<UP,BROADCAST,MULTICAST>  mtu 1500
        ether b8:27:eb:56:14:30  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 0  bytes 0 (0.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

lo: flags=73<UP,LOOPBACK,RUNNING>  mtu 65536
        inet 127.0.0.1  netmask 255.0.0.0
        loop  txqueuelen 1000  (Local Loopback)
        RX packets 10684825  bytes 3200716939 (2.9 GiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 10684825  bytes 3200716939 (2.9 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.142  netmask 255.255.255.0  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)
        RX packets 1202003  bytes 170234753 (162.3 MiB)
        RX errors 0  dropped 58321  overruns 0  frame 0
        TX packets 1584269  bytes 389475184 (371.4 MiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0

wlan0:0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.144  netmask 255.255.255.255  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)

wlan0:1: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.146  netmask 255.255.255.255  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)

wlan0:2: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.148  netmask 255.255.255.255  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)

wlan0:3: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.186  netmask 255.255.255.255  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)

wlan0:4: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.0.150  netmask 255.255.255.255  broadcast 192.168.0.255
        ether b8:27:eb:03:41:65  txqueuelen 1000  (Ethernet)
[/code]

but the problem is not solved alexa find devices but saying not respondig bridge version v5.4.0

what have i done 

update to v5.4.0
renumber devices 
let alexa search for new devices she found 11 devices but all not responding
